Description: Fix for lib names
 Upstream is not interested in renaming these shared libraries.
Author: Boris Pek <tehnick-8@yandex.ru>
Last-Update: 2013-09-13

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -136,7 +136,7 @@
 endif ()
 
 set (LEECHCRAFT_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
-set (LEECHCRAFT_LIBRARIES lcutil xmlsettingsdialog)
+set (LEECHCRAFT_LIBRARIES leechcraft-util leechcraft-xmlsettingsdialog)
 
 set (CMAKE_MODULE_PATH "/usr/local/share/apps/cmake/modules;/usr/share/apps/cmake/modules;${CMAKE_MODULE_PATH}")
 
--- a/src/FindLeechCraft.cmake.in
+++ b/src/FindLeechCraft.cmake.in
@@ -6,8 +6,8 @@
 if (LEECHCRAFT_INCLUDE_DIR)
 	set (LEECHCRAFT_INCLUDE_DIR "${LEECHCRAFT_INCLUDE_DIR}/leechcraft")
 endif ()
-find_library(_LEECHCRAFT_UTIL NAMES lcutil)
-find_library(_LEECHCRAFT_XSD NAMES xmlsettingsdialog)
+find_library(_LEECHCRAFT_UTIL NAMES leechcraft-util)
+find_library(_LEECHCRAFT_XSD NAMES leechcraft-xmlsettingsdialog)
 set (LEECHCRAFT_LIBRARIES ${_LEECHCRAFT_UTIL} ${_LEECHCRAFT_XSD})
 set (LEECHCRAFT_USE_FILE "${CMAKE_MODULE_PATH}/UseLeechCraft.cmake")
 
--- a/src/util/CMakeLists.txt
+++ b/src/util/CMakeLists.txt
@@ -116,20 +116,21 @@
 add_definitions(-DQT_NO_DEBUG)
 add_definitions(-DQT_SHARED)
 add_definitions(-DQT_GUI_LIB)
+add_definitions(-Dleechcraft-util_EXPORTS)
 
-add_library (lcutil SHARED
+add_library (leechcraft-util SHARED
 	${SRCS}
 	${UIS_H}
 	)
-target_link_libraries (lcutil
+target_link_libraries (leechcraft-util
 	${QT_LIBRARIES}
 	${MAGIC_LIBRARIES}
 	${ADDITIONAL_LIBS}
-	xmlsettingsdialog
+	leechcraft-xmlsettingsdialog
 	)
-set_property (TARGET lcutil PROPERTY SOVERSION 0.5.0)
-install (TARGETS lcutil DESTINATION ${LIBDIR})
+set_property (TARGET leechcraft-util PROPERTY SOVERSION 0.5.0)
+install (TARGETS leechcraft-util DESTINATION ${LIBDIR})
 
 if (USE_QT5)
-	QT5_USE_MODULES (lcutil Declarative Network Xml Sql Widgets)
+	QT5_USE_MODULES (leechcraft-util Declarative Network Xml Sql Widgets)
 endif ()
--- a/src/xmlsettingsdialog/CMakeLists.txt
+++ b/src/xmlsettingsdialog/CMakeLists.txt
@@ -7,6 +7,7 @@
 	${CMAKE_CURRENT_BINARY_DIR}
 	..
 	)
+add_definitions (-Dxmlsettingsdialog_EXPORTS)
 
 set (SRCS
     xmlsettingsdialog.cpp
@@ -111,16 +112,16 @@
 set (XSD_SRCS ${XSD_SRCS} PARENT_SCOPE)
 set (XSD_FORMS ${XSD_FORMS} PARENT_SCOPE)
 
-add_library (xmlsettingsdialog SHARED
+add_library (leechcraft-xmlsettingsdialog SHARED
 	${SRCS}
 	${MOC_SRCS}
 	${UIS_H})
-target_link_libraries (xmlsettingsdialog
+target_link_libraries (leechcraft-xmlsettingsdialog
 	${QT_LIBRARIES}
 	)
-set_property (TARGET xmlsettingsdialog PROPERTY SOVERSION 0.3.0)
-install (TARGETS xmlsettingsdialog DESTINATION ${LIBDIR})
+set_property (TARGET leechcraft-xmlsettingsdialog PROPERTY SOVERSION 0.3.0)
+install (TARGETS leechcraft-xmlsettingsdialog DESTINATION ${LIBDIR})
 
 if (USE_QT5)
-	QT5_USE_MODULES (xmlsettingsdialog Xml Sql Widgets Script Network)
+	QT5_USE_MODULES (leechcraft-xmlsettingsdialog Xml Sql Widgets Script Network)
 endif ()
